great_expectations/render/renderer_configuration.py

```diff
@@ -180,37 +167,148 @@ def __eq__(self, other: Any) -> bool:
             else:
                 return self == other
 
+    @staticmethod
+    def _get_renderer_param_base_model_type(name: str) -> Type[BaseModel]:
+        return create_model(
+            name,
+            renderer_schema=(
+                Dict[str, RendererSchemaType],
+                Field(..., alias="schema"),
+            ),
+            value=(Union[Any, None], ...),
+            __base__=RendererConfiguration._RendererParamBase,
+        )
+
+    @staticmethod
+    def _get_evaluation_parameter_params_from_raw_kwargs(
+        raw_kwargs: Dict[str, Any]
+    ) -> BaseModel:
```

Comment:
```suggestion
    ) -> _RendererParamsBase:
```
Same point as above.